src/components/address/AddressLookup/__tests__/AddressLookup-test.js

```diff
@@ -53,8 +53,8 @@ describe('AddressLookup', function() {
   it('accepts an existing address', function() {
     var element = TestUtils.renderIntoDocument(<AddressLookup address={ addressFindResult.address } />);
     var breakdown = findByClass(element, 'AddressBreakdown');
```

Comment:
Cheers, there might be a couple of these. Might see about eslint config to pick them up.